science/gnss-sdr/Portfile

```diff
@@ -220,20 +223,23 @@ if {![variant_isset opencl]} {
 
 variant osmosdr description "Install ${name} with support for OsmoSDR and other front-ends (RTL-based dongles, HackRF, bladeRF, etc.) as signal source." {
 
-    if {${subport} ne "gnss-sdr-next"} {
+    if {[catch {set installed [lindex [registry_active gnuradio-next] 0]}]} {
```

Comment:
After testing, this "if" isn't necessary any longer. Please just leave the internal code.

Reply:
Done